scripts/globals/abilities/feather_step.lua

```diff
@@ -36,74 +36,111 @@ function onUseAbility(player, target, ability, action)
     local effect = 1
 
     if math.random() <= getHitRate(player, target, true, player:getMod(tpz.mod.STEP_ACCURACY)) then
-        hit = 6	
+        hit = 6
+        local mjob = player:getMainJob()
         local daze = 1
 
-        if (target:hasStatusEffect(tpz.effect.BEWILDERED_DAZE_1)) then
-            local duration = target:getStatusEffect(tpz.effect.BEWILDERED_DAZE_1):getDuration()
-            target:delStatusEffectSilent(tpz.effect.BEWILDERED_DAZE_1)
-            if (player:hasStatusEffect(tpz.effect.PRESTO)) then
-                target:addStatusEffect(tpz.effect.BEWILDERED_DAZE_3, 1, 0, duration+30)
-                daze = 3
-                effect = 3
+        if (mjob == 83) then
```

Comment:
there is no job ID 83 (and we'd use the human readable name from status.lua anyway) so what is the intent here?

Reply:
I was in a rush this morning and wasn’t really thinking as I put the number for the job n dumb me was thinking about what lvl it unlocked at... will fix in a little bit.